Update Hello World post to work with vers 0.11.0 (#56)
bitcoindevkit/bitcoindevkit.org: content/blog/2020/hello-world.md

description: "Getting started using the BDK library in a very simple Rust project"
authors: 
    - Alekos Filini

>     - Claudio Ciardelli

Reviewer: For doing an update like this I don't think you should change the article date or add your name here, this should stay as the original author(s) and published date.  

But you should add a line at the end of the file to indicate that the article was updated for `bdk v0.11`, something like:

_Revised 2021-09 to support API changes in BDK version `v0.11.0`._

Author: Fixed

## Introduction

This article should serve as a "getting started" guide for developers who are considering integrating BDK in their projects: it tries to introduce the reader to the basic concepts behind the library and some of its

> modules and components that can be used to build a very simple functioning Bitcoin wallet. All the information written in this article are valid for the current `master` git branch, and should remain valid for the upcoming [`v0.11.0` release](https://github.com/bitcoindevkit/bdk/projects/1)

Reviewer: The `0.11.0` release is out now, and it's better to link to the cargo version since that's where folks will find the latest api docs. 

```suggestion
modules and components that can be used to build a very simple functioning Bitcoin wallet. All the information written in this article are valid for the current `master` git branch, and should remain valid for the [`v0.11.0` release](https://crates.io/crates/bdk/0.11.0).
```
And you can remove the below line about it being released soon.

Going back to our `Wallet`: given a descriptor and a `Database` we can build an "air-gapped", or "Offline" wallet: basically, a wallet that physically can't to connect to the Bitcoin network. It will still be able to generate addresses and
sign [PSBTs][PSBT], but with a greatly reduced attack surface because a sizable part of the code that handles the logic to synchronize with the network would be entirely omitted in the final executable binary.

This is how an offline `Wallet` can be created. Notice that we are using [`MemoryDatabase`][MemoryDatabase] as our `Database`. We'll get to that in a second.

```rust
use bdk::{
        Wallet,

>         wallet::AddressIndex::New

Reviewer: Missing a comma:
```suggestion
        wallet::AddressIndex::New,
```

```rust
// ...

>     let finalized = wallet.sign(&mut psbt, SignOptions::default())?;

Reviewer: To be consistent add the missing use statement:
```suggestion
    use bdk::SignOptions;

    let finalized = wallet.sign(&mut psbt, SignOptions::default())?;
```
